### PR TITLE
[DOCFIX] Fix some inaccurate description in docs

### PR DESCRIPTION
**Pull request by apc999**

### CODE REVIEW

File: docs/cn/Building-Alluxio-Master-Branch.md [hunk 1]
```diff
@@ -10,19 +10,19 @@ group: Resources
...
 
 {% include Building-Alluxio-Master-Branch/OutOfMemoryError.md %}
 
-若需要构建一个特定的版本Alluxio，例如{{site.ALLUXIO_RELEASED_VERSION}}，先执行`cd alluxio`，接着执行`git checkout v{{site.ALLUXIO_RELEASED_VERSION}}`。
+若需要编译一个特定的版本Alluxio，例如{{site.ALLUXIO_RELEASED_VERSION}}，先执行`cd alluxio`，接着执行`git checkout v{{site.ALLUXIO_RELEASED_VERSION}}`。
 
-Maven构建环境将自动获取依赖，编译源码，运行单元测试，并进行打包。如果你是第一次构建该项目，下载依赖包可能需要一段时间，但以后的构建过程将会快很多。
+Maven编译环境将自动获取依赖，编译源码，运行单元测试，并进行打包。如果你是第一次编译该项目，下载依赖包可能需要一段时间，但以后的编译过程将会快很多。
 
 一旦构建完成，执行以下命令启动Alluxio：
```
Comment (bf8086) [inline docs/cn/Building-Alluxio-Master-Branch.md]: `一旦构建完成` => `一旦编译完成`?

File: docs/cn/Building-Alluxio-Master-Branch.md [hunk 2]
```diff
@@ -10,19 +10,19 @@ group: Resources
 
 该指南介绍如何从头编译Alluxio。
 
-这部分内容的前提条件是你已安装[Java 7 (或以上)](Java-Setup.html)、[Maven](Maven.html)以及[Thrift 0.9.3](Thrift.html) (可选)。
+这部分内容的前提条件是你已安装[Java JDK 8或以上](Java-Setup.html)、[Maven 3.3.9或以上](Maven.html)以及[Thrift 0.9.3](Thrift.html) (可选)。
 
-从Github上获取主分支并打包：
+从Github上获取主分支：
```
Comment (bf8086) [inline docs/cn/Building-Alluxio-Master-Branch.md]: `获取主分支`=>`获取主分支并编译`?

The snippet below contains `$ mvn install -DskipTests`.

File: docs/en/Building-Alluxio-Master-Branch.md [hunk 3]
```diff
@@ -10,9 +10,9 @@ group: Resources
 
 This guide describes how to compile Alluxio from the beginning.
 
-The prerequisite for this guide is that you have [Java 7 (or above)](Java-Setup.html), [Maven](Maven.html), and [Thrift 0.9.3](Thrift.html) (Optional) installed.
+The prerequisite for this guide is that you have [Java 8 or later](Java-Setup.html), [Maven 3.3.9 or later](Maven.html), and [Thrift 0.9.3](Thrift.html) (Optional) installed.
 
-Checkout the Alluxio master branch from Github and package:
+Checkout the Alluxio master branch from Github:
```
Comment (bf8086) [inline docs/en/Building-Alluxio-Master-Branch.md]: `Checkout the Alluxio master branch from Github` => `Checkout the Alluxio master branch from Github and build`

File: docs/cn/Building-Alluxio-Master-Branch.md (hunk 2 above)
Comment (apc999) [reply docs/cn/Building-Alluxio-Master-Branch.md]: done

File: docs/cn/Building-Alluxio-Master-Branch.md (hunk 1 above)
Comment (apc999) [reply docs/cn/Building-Alluxio-Master-Branch.md]: done

File: docs/en/Building-Alluxio-Master-Branch.md (hunk 3 above)
Comment (apc999) [reply docs/en/Building-Alluxio-Master-Branch.md]: done

**Comment by apc999** [comment]
@bf8086  PTAL

**Pull request merged**